openapi/api.github.com/operations/apps/check-account-subscribes-to-plan.json

```diff
@@ -23,6 +23,18 @@
       "in": "path",
       "required": true,
       "schema": { "type": "integer" }
+    },
+    {
+      "name": "per_page",
+      "description": "Results per page (max 100)",
+      "in": "query",
+      "schema": { "type": "integer", "default": 30 }
+    },
+    {
+      "name": "page",
+      "description": "Page number of the results to fetch.",
+      "in": "query",
+      "schema": { "type": "integer", "default": 1 }
```

Comment:
Incorrect

Reply:
will be addressed with a follow up PR, won't cause much harm to merge it in, I don't want these changes to be blocked for too long